akd/src/tests.rs

```diff
@@ -108,16 +108,13 @@ async fn test_simple_lookup() -> Result<(), AkdError> {
     ])
     .await?;
     // Get the lookup proof
-    let lookup_proof = akd.lookup(AkdLabel::from_utf8_str("hello")).await?;
-    // Get the root hash with respect to which lookup_proof should verify
-    let current_azks = akd.retrieve_current_azks().await?;
-    let root_hash = akd.get_root_hash(&current_azks).await?;
+    let (lookup_proof, root_hash) = akd.lookup(AkdLabel::from_utf8_str("hello")).await?;
     // Get the VRF public key
     let vrf_pk = akd.get_public_key().await?;
     // Verify the lookup proof
     lookup_verify(
         vrf_pk.as_bytes(),
-        root_hash,
+        root_hash.1,
```

Comment:
Nit: `root_hash.hash()`, as well as maybe naming the variable `epoch_hash`

A few more instances of `root_hash.0` and `root_hash.1` in this file that can also be updated :)